### ISSUE TITLE
Missing tree and sapling from 2006

### ISSUE THREAD

**Issue by johnpauls** [NCRN/ForestVeg#47]
I was looking to see how many "missing" trees and saplings we have. I noticed that there was a tree and a sapling missing from 2006 , I did not expect that as these would have been sampled right after setup, but I guess 2 is not that many. Here is the deal:

Tag 458. a sapling from GWMP-0207 was missing in 2006, but Alive leaning in 2010 and Alive standing in 2014. 

Tag 304 is a tree from CHOH-0539. it was Missing in 2006, but Alive fallen in 2010 and Alive leaning in 2018.

In both cases we know that they were Alive in 2006 - should we go  back and correct that (and any other similar issues)?

**Comment by blcampbell** [comment]
@johnpauls, @lizmatthews03  We probably should discuss how best to document these adjustments for clarity on early data. That is for certified data how needed adjustments can be appropriately made so someone can follow the data.

**Comment by johnpauls** [comment]
I agree. We need to think through what we want the data to look like as this type of issue is found. One option is to leave it as is and let the end user go through and correct this, but this makes using the data more onerous and invites potential errors.

Another option is to maybe make a tree status like "Alive Missed" that can only be applied during QA/QC long after the sampling event happens and the tree/sapling is found alive. This would make it easy to use the data for densities, mortality estimates etc and also indicate why there is no dbh or other data.

However, it does make it complicated to know what to do if a plant is Alive in one cycle, Missed in the next and then Dead in the 3rd. In the second cycle it is like Schrödinger's tree - we don't know if it was alive or dead, so I am not sure what should be done there.

**Comment by blcampbell** [comment]
Ref: [Schrödinger's tree](http://strangehorizons.com/poetry/schrdingers-tree/)
